tutorials/post_processing.md

```diff
@@ -0,0 +1,15 @@
+## Simulation Output Post-processing
+
+Each time a HIVpy model run is completed, some basic post-processing is run at the end to produce graphs of the column contents specified by a `graph_outputs` list under `EXPERIMENT` in the model YAML config file. Any valid column name in `graph_outputs` will be used for plotting.
+
+### Create an Average of Multiple Simulation Runs
+
+Once you have multiple CSV output files from multiple model runs, ensure they are all in one directory. This will be your input directory for data aggregation.
+
+Next, run post-processing from the terminal with the following command:
+```bash
+python <path>/<to>/post_processing.py <configuration file> <input directory> <output directory>
+```
```

Comment:
I forgot about this earlier, but the usage section needs to be updated to account for the graphing option integration changes made in [c46b9f3](https://github.com/UCL/hivpy/pull/176/commits/c46b9f3b7e06496cbd97f548ae699c08b2e474d2).